radioactive-decay/info.yml

```diff
@@ -0,0 +1,14 @@
+---
+uid: radioactive_decay 
+title: radioactive decay 
+prerequisites:
+  - undergraduate statistics and probability
+learning_objectives:
+  - basic understanding of radioactivity
```

Comment:
(continuing to use this as a way to discuss some of the issues about scope, etc, ...)
This continues to seem like a broad learning objective.

Looking at the content, it seems that the focus is on the time behavior of radioactive decay, and none of the nuclear physics.  I think the scope is probably what I would expect, but the statement of learning objective does not match that scope.  Might also need some additional pre-requisites.

Interesting question on pre-requisites: Some instructors may choose different orders (deductive vs inductive style) and thus indicate pre-requesites in a completely different order.  For example, related to this node - some instructors might choose to discuss some aspects of decay (types of decay) before this node and others might do it after this node.